### PR TITLE
Add Toit LSP

### PR DESCRIPTION
**Pull request by serjan-nasredin**
Hello, Toit LSP for Sublime.

### CODE REVIEW

File: docs/src/language_servers.md [hunk 1]
```diff
@@ -717,6 +717,11 @@ Follow installation instructions on [LSP-tailwindcss](https://github.com/sublime
 
 Follow installation instructions on [LSP-terraform](https://github.com/sublimelsp/LSP-terraform).
 
+## Toit
+
+Install the [Toit](https://packagecontrol.io/packages/Toit) package from Package Control for syntax highlighting.
+Follow installation instructions on [LSP-toit](https://github.com/toitlang/toit/tree/master/tools/lsp/server).
```
Comment (predragnikolic) [inline docs/src/language_servers.md:723]: There are no installation instructions at https://github.com/toitlang/toit/tree/master/tools/lsp/server , you maybe put the wrong link.

File: docs/src/language_servers.md [hunk 2]
```diff
@@ -717,6 +717,11 @@ Follow installation instructions on [LSP-tailwindcss](https://github.com/sublime
...
+
```
Comment (predragnikolic) [inline docs/src/language_servers.md:724]: you are missing the 3rd step, and that is configuration that a user needs to setup. Look at other examples in the language_servers.md, it usually stays with:

3. Open `Preferences > Package Settings > LSP > Settings` and add the `"typst-lsp"` client configuration to the `"clients"`:
```
// The confirmation 
```

**Comment by serjan-nasredin** [comment]
@predragnikolic, can you see the change?

**Review by predragnikolic — APPROVED** [review]

**Comment by predragnikolic** [comment]
Thanks, 
sorry for the wait :)

File: docs/src/language_servers.md [hunk 3]
```diff
@@ -717,6 +717,24 @@ Follow installation instructions on [LSP-tailwindcss](https://github.com/sublime
 
 Follow installation instructions on [LSP-terraform](https://github.com/sublimelsp/LSP-terraform).
 
+## Toit
+
+1. Install the [Toit](https://packagecontrol.io/packages/Toit) package from Package Control for syntax highlighting.
+2. Install the [Jaguar](https://github.com/toitlang/jaguar)
```
Comment (predragnikolic) [inline docs/src/language_servers.md:723]:
```suggestion
2. Install the [Jaguar Language Server](https://github.com/toitlang/jaguar)
```

**Comment by serjan-nasredin** [comment]
> Thanks, sorry for the wait :)

No problem :) ❤️

**Review by predragnikolic — APPROVED** [review]

**Comment by predragnikolic** [comment]
Thanks again

**Pull request merged**